### PR TITLE
Change of logic in function deploy_app and mandatory mandatory_aetest_arguments

### PR DESCRIPTION
**Pull request by GoshkaEgoshka**
In the function deploy_app, the loading of the testbed file was removed.
In the mandatory_aetest_argumets was added device. This helped pass the device
 argument to deploy_app.

### CODE REVIEW

File: oct/tests/__init__.py [hunk 1]
```diff
@@ -6,7 +6,11 @@
 
 
 def mandatory_aetest_arguments(testbed: Testbed,) -> Dict[str, Any]:
-    return {"testbed": testbed, "grid": testbed.custom["selenium-grid"]}
+    return {
+        "testbed": testbed,
+        "grid": testbed.custom["selenium-grid"],
+        "device": testbed.devices["vm"],
```
Comment (extsoft) [inline oct/tests/__init__.py]: @GoshkaEgoshka please make device name configurable via methods parameters. I may want to change it in the test without modifying this method.

Comment (GoshkaEgoshka) [reply oct/tests/__init__.py]: @extsoft I fixed it. Please, review.

File: suite.py [hunk 2]
```diff
@@ -28,7 +29,7 @@ def main(runtime: EasypyRuntime) -> None:
                 )
             ),
             testscript=full_test_path,
-            **mandatory_aetest_arguments(runtime.testbed),
+            **mandatory_aetest_arguments(runtime.testbed, device_name="vm"),
```
Comment (extsoft) [inline suite.py]: @GoshkaEgoshka Please do not specify default value again.

Comment (extsoft) [reply suite.py]: @GoshkaEgoshka I lost a context here. The code is correct.

**Pull request merged**